diff --git a/src/humanize/time.py b/src/humanize/time.py
index 9d43d00..60c0670 100644
--- a/src/humanize/time.py
+++ b/src/humanize/time.py
@@ -154,13 +154,14 @@ def naturaldelta(
 
     use_months = months
 
-    delta = abs(delta)
-    years = delta.days // 365
-    days = delta.days % 365
+    seconds = abs(delta.seconds)
+    days = abs(delta.days)
+    years = days // 365
+    days = days % 365
     num_months = round(days / 30.5)
 
     if years == 0 and days < 1:
-        if delta.seconds == 0:
+        if seconds == 0:
             if min_unit == Unit.MICROSECONDS and delta.microseconds < 1000:
                 return (
                     _ngettext("%d microsecond", "%d microseconds", delta.microseconds)
@@ -177,14 +178,14 @@ def naturaldelta(
                 )
             return _("a moment")
 
-        if delta.seconds == 1:
+        if seconds == 1:
             return _("a second")
 
-        if delta.seconds < 60:
-            return _ngettext("%d second", "%d seconds", delta.seconds) % delta.seconds
+        if seconds < 60:
+            return _ngettext("%d second", "%d seconds", seconds) % seconds
 
-        if 60 <= delta.seconds < 3600:
-            minutes = round(delta.seconds / 60)
+        if 60 <= seconds < 3600:
+            minutes = round(seconds / 60)
             if minutes == 1:
                 return _("a minute")
 
@@ -193,8 +194,8 @@ def naturaldelta(
 
             return _ngettext("%d minute", "%d minutes", minutes) % minutes
 
-        if 3600 <= delta.seconds:
-            hours = round(delta.seconds / 3600)
+        if 3600 <= seconds:
+            hours = round(seconds / 3600)
             if hours == 1:
                 return _("an hour")
 
